alvr/client_openxr/src/lib.rs

```diff
@@ -118,7 +118,7 @@ pub fn entry_point() {
 
     let loader_suffix = match platform {
         Platform::Quest1 => "_quest1",
-        p if p.is_pico() => "_pico",
+        Platform::PicoNeo3 => "_pn3",
```

Comment:
are we sure we don't want to do it for unknown too?

Reply:
yeah, thanks

Reply:
There's no older pico devices that alvr runs on that we're not keeping track of?

Reply:
No others that run OpenXR